.github/dependabot.yml

```diff
@@ -11,5 +11,3 @@ updates:
       interval: "daily"
       time: "09:00"
       timezone: "America/New_York"
-    # Allow up to 10 open pull requests for dependencies
-    open-pull-requests-limit: 10
```

Comment:
Is this using the default?  Is it 1?

Reply:
Default is 5
https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#open-pull-requests-limit